src/commcare_cloud/ansible/deploy_common.yml

```diff
@@ -17,14 +17,14 @@
   tags: common
 
 - name: ufw (firewall)
-  hosts: 'all_commcarehq:!proxy:!cas_proxy:!pna_proxy:!couchdb2_proxy:!reach_proxy'
+  hosts: 'all_commcarehq:!proxy:!cas_proxy:!couchdb2_proxy:!reach_proxy'
```

Comment:
I assume `cas_proxy` is also obsolete. Same below.

Reply:
Yes, good call. I missed that on #6358